third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -1991,7 +1991,10 @@ resource "google_container_cluster" "primary" {
     }
     gce_persistent_disk_csi_driver_config {
       enabled = false
-    }
+	}
```

Comment:
nit: be careful about tabs vs spaces in test HCL. Either is technically allowed, but we should be consistent within a test (and I believe we use spaces more often since that's what `terraform fmt` does to regular tf configs)